src/pages/Events/SingleEvent/__test__/SingleEvent-buttons.test.js

```diff
@@ -78,7 +39,8 @@ describe("Test SingleEvent component functionality", () => {
   it.skip("should have share button working correctly", () => {});
```

Comment:
Did you use .skip to skip yet not implemented tests and to implement this in further work? I didn't notice this in previous PR's and I'm curious 😄

Reply:
Yes, this is because we haven't figure out yet how the share button should work. Once I get a ticket describing how the functionality will be I will implemented, and once done I will do the test :)